sass/sass.bzl

```diff
@@ -41,8 +41,10 @@ def _sass_binary_impl(ctx):
 
     # Load up all the transitive sources as dependent includes.
     transitive_sources = collect_transitive_sources(ctx)
-    for src in transitive_sources:
-        options += ["-I={0}".format(src)]
+    transitive_source_folders = set([src.path[:-len(src.basename)] for src in transitive_sources])
```

Comment:
Oh nice. That would be way cleaner.

My current problem is that setting `-I=path` actually doesn't fix anything.